packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1658,6 +1658,15 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
   @override
   TextEditingValue get currentTextEditingValue => _value;
 
+  @override
+  void updateEditingValueWithDeltas(List<TextEditingDelta> textEditingDeltas) {
+    TextEditingValue value = _value;
+    for (final TextEditingDelta delta in textEditingDeltas) {
+      value = delta.apply(value);
+    }
+    updateEditingValue(value);
```

Comment:
Smart, just call through to updateEditingValue...  For a rich text editor author, could they still disable this?

Reply:
A rich text editor author at this low level I think would just implement `updateEditingValueWithDeltas` themselves.